go.mod

```diff
@@ -1,10 +1,9 @@
 module github.com/consensys/quorum-key-manager
 
-go 1.15
+go 1.16
```

Comment:
is there any special need from bumping minimum go version?

Reply:
reverted